From ed7f981215a41d689cfbf8c25749e24b62571558 Mon Sep 17 00:00:00 2001
From: "E. Dunham" <dunhame@onid.orst.edu>
Date: Mon, 14 Apr 2014 11:05:41 -0700
Subject: [PATCH] Update testing with how Trial finds tests

For ticket #4526. Added link to documentation on where trial's tests
come from.
---
 docs/core/howto/testing.rst | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/docs/core/howto/testing.rst b/docs/core/howto/testing.rst
index d715045..05e1da2 100644
--- a/docs/core/howto/testing.rst
+++ b/docs/core/howto/testing.rst
@@ -20,8 +20,8 @@ Trial basics
 library for writing test cases and utility functions for working with the
 Twisted environment in your tests, and a command-line utility for running your
 tests. Trial is built on the Python standard library's ``unittest`` 
-module.
-
+module. For more information on how Trial finds tests, see the
+:api:`twisted.trial.runner.TestLoader.loadModule <loadModule>` documentation.
 
 
 
-- 
1.9.0

